Tests/FunctionalTests/Sources/FTRSwiftTests.swift

```diff
@@ -69,6 +69,26 @@ class FunctionalTestRigSwiftTests: XCTestCase {
         .assertWithMatcher(grey_nil())
   }
 
+  func testInteractionWithALabelWithParentHidden() {
+    let checkHiddenBlock:GREYActionBlock =
+        GREYActionBlock.actionWithName("checkHiddenBlock", performBlock: { element, errorOrNil in
+                                        // Check if the found element is hidden or not.
+                                        let superView:UIView! = element as! UIView
```

Comment:
Ouch. That might cause issues with our internal style guide. Right now I'm basing this similarly to our ObjC one.